sources/packages/api/src/esdc-integration/e-cert-integration/models/e-cert-integration-model.ts

```diff
@@ -78,3 +78,8 @@ export interface ECertUploadResult {
   generatedFile: string;
   uploadedRecords: number;
 }
+
+export interface ESDCFileResponseDTO {
```

Comment:
Please avoid keeping the DTO here to avoid confusion with the controller's DTOs.

Reply:
removed 'DTO`